item.rb

```diff
@@ -15,15 +15,11 @@ def can_be_archived?
   end
 
   def move_to_archive
-    @archived = true if can_be_archived? == true
+    @archived = can_be_archived?
```

Comment:
I don't know why you have changed this but the first approach is more appropriate. 
**@archived = true if can_be_archived? == true** This expression assigns true to @archived only if the result of can_be_archived? is true. If can_be_archived? returns false, @archived will not be assigned a value. This is more appropriate in our case because @archived is by default false so we don't need the reassignment.
**@archived = can_be_archived?**
This expression assigns the result of can_be_archived? directly to @archived. Whatever value can_be_archived? returns, whether true or false, will be assigned to @archived. In most cases, this approach is preferred but in our case, since we have a default for the property the first approach is preferred.

Reply:
Hi @kifle23, Thanks for the constructive feedback, Well I have changed this line due to unit test errors but as you said the previous line is more appropriate then I will update the code with the previous one.